app/javascript/packs/articlePage.jsx

```diff
@@ -75,10 +75,8 @@ document
   ?.addEventListener('click', copyArticleLink);
 
 // Comment Subscription
-const userDataIntervalID = setInterval(async () => {
+getCsrfToken().then(async () => {
```

Comment:
This change makes sure that logged-in status is properly available in the test environment to prevent Cypress flake - similar to the issues described in #13228. I haven't used the other helper mentioned in that issue (`getUserDataAndCsrfToken`) because this button is initialized even when userData is null (user is logged out).

I suspect this may also be related to #12545.